src/cms/views/events/event_view.py

```diff
@@ -181,17 +181,17 @@ def post(self, request, **kwargs):
                 if published:
                     messages.success(
                         request,
-                        _("Event translation was successfully created and published."),
+                        _("Translation event was successfully created and published"),
```

Comment:
I don't think "translation event" is the correct term here. I think this creates the impression that this is about an event with a "translation" topic...
I'd suggest just use the message "Event was successfully created and published" no matter if `event_instance` is `None` or not.

Reply:
Fixed.